targets/grandcentral-m4.json

```diff
@@ -0,0 +1,11 @@
+{
+    "inherits": ["atsamd51p20a"],
+    "build-tags": ["grandcentral_m4"],
+    "flash-1200-bps-reset": "true",
+    "flash-method": "msd",
+    "msd-volume-name": "GCM4BOOT",
+    "msd-firmware-name": "firmware.uf2",
+    "openocd-transport": "swd",
+    "openocd-interface": "jlink",
+    "openocd-target": "atsame5x"
```

Comment:
Not necessary since it is defined in cortex-m4.json

Reply:
Thanks, will remove